packages/wallets/src/components/Dropzone/Dropzone.scss

```diff
@@ -0,0 +1,89 @@
+.wallets-dropzone {
+    display: flex;
+    padding: 2.8rem 15.2rem 3.5rem;
+    justify-content: center;
+    align-items: center;
+    border-radius: 8px;
```

Comment:
how about using `BORDER_RADIUS_2`?

Reply:
Same as above, we need to create variables for border radius as well 😓